### PR TITLE
feat: added 'similar collections' to graasp explore

### LINKED ISSUE #7: add similar collections section

### PR DESCRIPTION
**Pull request by hagoptaminian**
I added dummy 'similar collection' data to src/data/sample, which I used in a new 'Similar Collections' section at the bottom of the main Graasp Explore Collections page. The 'Similar Collections' section I created follows the component pattern of other sections of the application (a SimilarCollections component which renders a SimilarCollectionsHeader and individual SimilarCollection components).

closes #7

### CODE REVIEW

**Comment by juancarlosfarah** [comment]
Thanks @hagoptaminian! Can you add a screenshot? Also, just as a sidenote, we often use “feat” instead of “improvement” on the commit messages. Thanks!

**Comment by hagoptaminian** [comment]
Absolutely! Here is a screenshot of the section I added:

Screen Shot 2020-05-02 at 3 10 38 PM

File: src/components/collection/SimilarCollection.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import SimilarCollectionBadges from './SimilarCollectionBadges';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 0,
+    paddingTop: '56.25%', // 16:9
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollection.js:26]: Where did you get this from? Do we really need this padding?

File: src/components/collection/SimilarCollection.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import SimilarCollectionBadges from './SimilarCollectionBadges';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 0,
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollection.js:25]: What does the height `0` do?

File: src/components/collection/SimilarCollection.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    marginLeft: 'auto',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  expandOpen: {
+    transform: 'rotate(180deg)',
+  },
+  avatar: {
+    backgroundColor: red[500],
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollection.js:39]: Are you using this class?

File: src/components/collection/SimilarCollection.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        avatar={<Avatar aria-label="recipe" src={creator.avatar} />}
+        action={
+          <IconButton aria-label="settings">
+            <MoreVertIcon />
+          </IconButton>
+        }
+        title={name}
+        subheader={`a collection by ${creator.name}`}
+      />
+      <CardMedia className={classes.media} image={image} title="Paella dish" />
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollection.js:72]: Make sure you also make the `title` dynamic. Otherwise you have `Paella dish` for everything.

Comment (juancarlosfarah) [reply src/components/collection/SimilarCollection.js:72]: You can use the name of the collection, for example.

File: src/components/collection/SimilarCollection.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <Card className={classes.root}>
+      <CardHeader
+        avatar={<Avatar aria-label="recipe" src={creator.avatar} />}
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollection.js:63]: Make sure you make `aria-label` dynamic. You can use the name of the creator, for example.

File: src/components/collection/SimilarCollection.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    name,
+    image,
+    description,
+    creator,
+    views,
+    likes,
+    rating,
+  } = similarCollection;
+  const classes = useStyles();
+  const [expanded, setExpanded] = React.useState(false);
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollection.js:54]: Are you using this `expanded` state attribute?

File: src/components/collection/SimilarCollections.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <SimilarCollectionsHeader />
+      {!similarCollections.length ? (
+        <div className="Main">
+          <Typography variant="h5" color="inherit">
+            {t('This collection is empty.')}
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollections.js:31]: You would want to change the text to say something like "No similar collections available."

File: src/components/collection/SimilarCollections.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+            >
+              <SimilarCollection similarCollection={similarCollection} />
+            </Grid>
+          ))}
+        </Grid>
+      )}
+    </div>
+  );
+}
+
```
Comment (juancarlosfarah) [inline src/components/collection/SimilarCollections.js:54]: This should be adapted to the collections, not left commented out.

Comment (juancarlosfarah) [reply src/components/collection/SimilarCollections.js:54]: Doesn't `yarn lint` complain that the prop types are not set correctly?

File: src/components/collection/SimilarCollection.js (hunk 6 above)
Comment (hagoptaminian) [reply src/components/collection/SimilarCollection.js:54]: You're right. This is not needed. I removed it.

File: src/components/collection/SimilarCollection.js (hunk 2 above)
Comment (hagoptaminian) [reply src/components/collection/SimilarCollection.js:25]: I manipulated this to see its effects, and as it stands it's not needed. The image in CardMedia will use a default height that I don't think it's necessary to manipulate.

File: src/components/collection/SimilarCollections.js (hunk 8 above)
Comment (hagoptaminian) [reply src/components/collection/SimilarCollections.js:54]: You're right, `lint` does complain. I commented out this section initially and forgot to uncomment it. Let's discuss the purpose of .propTypes and how it works.

File: src/components/collection/SimilarCollection.js (hunk 1 above)
Comment (hagoptaminian) [reply src/components/collection/SimilarCollection.js:26]: This is the default style in the Card component I used. The padding is needed: eliminating it causes the image embedded in Card to shrink vertically.

**Comment by juancarlosfarah** [comment]
@hagoptaminian, let me know when this is ready to merge.

**Comment by juancarlosfarah** [comment]
@hagoptaminian, can you include a small screenshot of the menu and how it looks on a similar collection card?

File: src/components/collection/SimilarCollection.js [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+import SimilarCollectionBadges from './SimilarCollectionBadges';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    paddingTop: '56.25%', // 16:9
+  },
+  avatar: {
```
[the remark this answers is not in the document]
Comment (hagoptaminian) [reply src/components/collection/SimilarCollection.js:25]: Removed!

**Comment by hagoptaminian** [comment]
> @hagoptaminian, can you include a small screenshot of the menu and how it looks on a similar collection card?

Screen Shot 2020-05-05 at 3 56 16 PM

**Pull request merged**